proposals/NNNN-disallow-inout-explicit-convs.md

```diff
@@ -0,0 +1,112 @@
+---
+title: "NNNN - Disallow all explicit conversions in inout parameters"
```

Comment:
Consider changing the proposal to be more general, like "Disallow all explicit conversions from producing an LValue"